cl/scrapers/tasks.py

```diff
@@ -54,12 +54,13 @@ def update_document_from_text(
 
     :param opinion: Opinion object
     :param juriscraper_module: full module to get Site object
-    :return: None
+    :return: the extracted data dictionary
     """
     court = opinion.cluster.docket.court.pk
     site = get_scraper_object_by_name(court, juriscraper_module)
     if site is None:
-        return
+        logger.debug("No site found %s", juriscraper_module)
+        return {}
```

Comment:
Perhaps we should just validate our modules so we dont end up here?

Reply:
I just added a `logger.debug`, but that `if site is None` is there from before this PR.  I think trying to track all the cases where that condition may happen is out of the scope of this PR